### ISSUE TITLE
Contao 5.3 Error 

### ISSUE THREAD

**Issue by simonla82** [Tastaturberuf/contao-image-copyright-bundle#13]
Thank you so much for porting the bundle to Contao 5.*! I just installed it and am very happy to be able to use it.
However, when I try to create a frontend module "Image-Copyright-Liste", I get the following eror message:

```
Tried to list template options for the modern fragment type "frontend_module/image_copyright_list" but could not find any template. In case you wanted to use the legacy type "mod_image_copyright_list", define it explicitly in the "template" property of your controller's service tag/attribute.
```

How can I circumvent that error?

**Comment by Tastaturberuf** [comment]
honestly is do not try it on 5.3 personaly. It was a simple PR from a
contributor with the comment that it works. i will look into it tomorrow.

**Comment by Tastaturberuf** [comment]
Fixed in  `1.3.1`
